data/tribes/initialization/amazons/units.lua

```diff
@@ -229,7 +229,7 @@ descriptions:new_tribe {
             helptexts = {
                purpose = {
                   -- TRANSLATORS: Helptext for an amazon ware: Ration, part 1
-                  pgettext("amazons_ware", "A small bite to keep miners strong and working. The scout also consumes rations on his scouting trips."),
+                  pgettext("amazons_ware", "A small bite to keep the workers digging for gold dust and granite strong and working. The scout also consumes rations on his scouting trips."),
```

Comment:
If Amazones are female this should be: "..consumes rations on her scouting trips."

Reply:
ooops. You are right